Framework/Geometry/inc/MantidGeometry/Rendering/Renderer.h

```diff
@@ -0,0 +1,120 @@
+#ifndef MANTID_GEOMETRY_RENDERER_H_
+#define MANTID_GEOMETRY_RENDERER_H_
+
+#include "MantidGeometry/DllConfig.h"
+#include "MantidGeometry/Rendering/OpenGL_Headers.h"
+#include <vector>
+
+class TopoDS_Shape;
+
+namespace Mantid {
+namespace Kernel {
+class V3D;
+}
+namespace Geometry {
+class RectangularDetector;
+class StructuredDetector;
+class IObjComponent;
+
+namespace detail {
+class GeometryTriangulator;
+}
+
+/** Renderer : Handles rendering details of geometry within mantid.
```

Comment:
Could probably say a bit more about this. Does not handle rendering of geometry concerned to md workspaces for example,  this is concerned with objects of geometry.